eng/pipelines/runtime-official.yml

```diff
@@ -481,6 +481,10 @@ extends:
       #
       # Build Workloads
       #
+      # note: if this is changed to use evaluatePaths job,
```

Comment:
the official build doesn't do path evaluation by design so I don't think we need this comment here